tests/unit/test_utils.py

```diff
@@ -93,3 +96,13 @@ def test_can_open_in_read(self):
             f.seek(0)
             assert f.read() == "foobar"
         self.assertFalse(os.path.isfile(filename))
+
+
+strings = ['', u'', 'some text', u'some text', 'yu\u3086', u'yu\u3086']
+
+
+@pytest.mark.parametrize('text', strings)
```

Comment:
It would probably be good to test more than just strings. Like dictionaries and lists as well to cover all of your bases.